### PR TITLE
fix: show icon button on mobile size

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

image

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectDataButtonGroup.vue [hunk 1]
```diff
@@ -1,9 +1,12 @@
 <script setup lang="ts">
+import { useWindowSize } from '@vueuse/core';
```
Comment (piggggggggy) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectDataButtonGroup.vue:2]: praise: vueuse can do everything..

**Review by piggggggggy — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue [hunk 2]
```diff
@@ -1,19 +1,9 @@
...
-                          @click="handleClickEdit"
-                >
-                    {{ $t('INVENTORY.COLLECTOR.DETAIL.EDIT') }}
-                </p-button>
-            </template>
-        </p-heading>
+        <section-header :title="$t('INVENTORY.COLLECTOR.DETAIL.BASE_INFO')"
+                        :edit-mode="state.isEditMode"
+                        @click-edit="handleClickEdit"
+        />
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-detail/modules/CollectorBaseInfoSection.vue:6]: praise: very simple!!!

**Review by yuda110 — APPROVED** [review]
LGTM! I like this change

**Pull request merged**